.azure-pipelines/azure-pipelines.yml

```diff
@@ -103,8 +103,6 @@ stages:
           targets:
             - name: CentOS 7
               test: centos7
-            - name: Fedora 35
-              test: fedora35
```

Comment:
How about replacing Fedora 34 in Docker_2_13 with Fedora 35? Right now you're testing Fedora 34 in both Docker_2_13 and Docker_2_12, but Fedora 35 would not be tested anymore when removed here.

Reply:
vgp!

i'll do that. thx @felixfontein